admin-dev/themes/new-theme/webpack.config.js

```diff
@@ -155,26 +167,28 @@ let config = {
   ]
 };
 
-// if (process.env.NODE_ENV === 'production') {
-//   config.plugins.push(
-//     new webpack.optimize.UglifyJsPlugin({
-//       sourceMap: false,
-//       compress: {
-//         sequences: true,
-//         conditionals: true,
-//         booleans: true,
-//         if_return: true,
-//         join_vars: true,
-//         drop_console: true
-//       },
-//       output: {
-//         comments: false
-//       }
-//     })
-//   );
-// } else {
+console.log(process.env.NODE_ENV);
+
+if (process.env.NODE_ENV === 'production') {
+  config.plugins.push(
+    new webpack.optimize.UglifyJsPlugin({
+      sourceMap: false,
+      compress: {
+        sequences: true,
+        conditionals: true,
+        booleans: true,
+        if_return: true,
+        join_vars: true,
+        drop_console: true
+      },
+      output: {
+        comments: false
```

Comment:
While we are on that subject, didn't you want to keep the license in the minimized files?

Reply:
Good point!
I don't want to keep all comments though, I'll try something else